firefox-ios/Client/Assets/CC_Script/FormAutofillChild.ios.sys.mjs

```diff
@@ -110,6 +110,13 @@ export class FormAutofillChild {
       return;
     }
 
+    // Since iOS doesn't support cross frame autofill,
+    // we should only call the autofill callback if the section is valid.
+    // TODO(issam): This will change when we have cross frame fill support.
+    if (!this.activeSection.isValidSection()) {
```

Comment:
mini nit: can do you have a TODO ticket for this comment

Reply:
Sure thing. I will need to do it in central first. Will open a patch for that and it should be eventually synced in here.